resources/CMakeLists.txt

```diff
@@ -87,6 +88,7 @@ add_packfile(dashfaction.vpp
     meshes/Vat1.v3m
     meshes/coffeesmokedtbl2.v3m
     meshes/coffeesmokedtblAlt.v3m
+    meshes/df_meatchunks0.v3m
```

Comment:
I can see that PS2 build uses `meatchunk1-5.v3d` and selects random ones and spawns 7 objects using `debris_create` function.
Is the mesh and texture copied from there? Shouldn't we make it the same and use multiple meshes?

Reply:
I'll replicate this behaviour. Perhaps spawning more chunks could also be considered, given the number 7 was probably picked due to performance considerations that are _far_ less relevant today than they would have been ~25 years ago. Will have to see how it looks.